### PR TITLE
[0.3.x] CAL-353 Handles non-UTF8 country codes

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Handles a country code that may not be UTF-8 encoded by detecting its encoding.
Uses : https://github.com/codice/ddf/pull/2426

#### Who is reviewing it? 
@brendan-hofmann @glenhein 
@ahoffer @mcalcote @garrettfreibott 

#### Choose 2 committers to review/merge the PR.
@rzwiefel 
@lessarderic

#### What are the relevant tickets?
[CAL-353](https://codice.atlassian.net/browse/CAL-353)

#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

File: catalog/video/video-mpegts-stream/pom.xml [hunk 1]
```diff
@@ -25,6 +25,23 @@
     <artifactId>video-mpegts-stream</artifactId>
     <name>Alliance :: Video :: Stream</name>
     <packaging>bundle</packaging>
+
+    <dependencyManagement>
```
Comment (lessarderic) [inline catalog/video/video-mpegts-stream/pom.xml]: ❓ Did we have to use `<dependencyManagement>` to get this to work? Could we have used a simple `<dependency>` without a `<version>`, scoped for test and with the `<exclusion>` instead?

**Comment by vinamartin** [comment]
build now

**Pull request merged**